tests/FunctionalTestCase.php

```diff
@@ -147,6 +150,78 @@ protected function assertCollectionCount($namespace, $count): void
         $this->assertEquals($count, $document['n']);
     }
 
+    /**
+     * Asserts that a collection with the given name does not exist on the
+     * server.
+     *
+     * If $databaseName is not specified, the current database will be used.
+     *
+     * @param string $collectionName
+     * @param string $databaseName
```

Comment:
I believe PHPDoc understands `?string`, but since that syntax was only introduced with PHP 7.1, many people still prefer `string|null`. I have no preference for either since I prefer using an argument type directly and skipping the `@param` annotation since the type information is already in the signature.